crates/config/tests/duplicate_preset.toml

```diff
@@ -12,11 +12,6 @@ ref = "default"
 plugins = ["blank_slate"]
 
 [[resource]]
-route = "/"
-plugins = ["blank_slate"]
-timeout = 25
-
-[[resource]]
-route = "/*params"
+routes = ["/"]
```

Comment:
The configuration files within the `crates/config/tests` directory are part of a test case. They don't need to be documented as their purpose is clear. They may contain errors intentionally since their purpose is to confirm correct handling in other parts of the codebase.